src/Core/TestSolution/TestSolution.TestProject/SdkClassLibrary1/SdkClassLibrary1.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TheUniversalAnswer>42</TheUniversalAnswer>
+        <Vegetable>Potato</Vegetable>
+        <Vegetable>Carrot</Vegetable>
+    </PropertyGroup>    
+    
+    <PropertyGroup>
+        <TargetFramework>netstandard2.1</TargetFramework>
+    </PropertyGroup>
+    
+    <PropertyGroup>
+        <WarningsAsErrors>CS0162</WarningsAsErrors>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="coverlet.collector" Version="6.0.0">
+        <PrivateAssets>all</PrivateAssets>
+        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      </PackageReference>
```

Comment:
**remark**: Let me know if you're unhappy with this package reference (in the test project) for some reason.

Reply:
This is fine 👍